src/ctap/storage.rs

```diff
@@ -525,29 +530,33 @@ impl PersistentStore {
 
     pub fn attestation_private_key(
         &self,
-    ) -> Result<Option<&[u8; ATTESTATION_PRIVATE_KEY_LENGTH]>, Ctap2StatusCode> {
+    ) -> Result<Option<&[u8; key_material::ATTESTATION_PRIVATE_KEY_LENGTH]>, Ctap2StatusCode> {
         let data = match self.store.find_one(&Key::AttestationPrivateKey) {
             None => return Ok(None),
             Some((_, entry)) => entry.data,
         };
-        if data.len() != ATTESTATION_PRIVATE_KEY_LENGTH {
+        if data.len() != key_material::ATTESTATION_PRIVATE_KEY_LENGTH {
```

Comment:
Why not import those constants? The name seems unambiguous enough that `key_material` is not needed.

Reply:
I could. Usually for constants I like knowing where they come from without looking at the top :D

Reply:
Ah ok, it's not too disambiguate but just to underline the definition origin.